test/Elastic.Apm.Tests/LabelTests.cs

```diff
@@ -0,0 +1,287 @@
+﻿// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using Elastic.Apm.Api;
+using Elastic.Apm.Tests.Mocks;
+using FluentAssertions;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Elastic.Apm.Tests
+{
+	public class LabelTests
```

Comment:
The tests should use the payload serializer instead of `JsonConvert.SerializeObject()`, It better represents the agent's serialization and means the tests don't need to change when swapping to a different serializer in the future

Reply:
Makes sense. Turns out we have `SerializationTests.cs` which does exactly that - now `LabelTests.cs` does it as well.